httpx/api.py

```diff
@@ -32,6 +32,51 @@ def request(
     stream: bool = False,
     trust_env: bool = None,
 ) -> Response:
+    """
+    Sends an HTTP request.
+
+    **Parameters:**
+
+    * **method** - HTTP method for the new `Request` object: `GET`, `OPTIONS`,
+    `HEAD`, `POST`, `PUT`, `PATCH`, or `DELETE`.
+    * **url** - URL for the new `Request` object.
+    * **params** - *(optional)* Query parameters to include in the URL, as a
+    string, dictionary, or list of two-tuples.
+    * **data** - *(optional)* Data to include in the body of the request, as a
+    dictionary
+    * **files** - *(optional)* A dictionary of upload files to include in the
+    body of the request.
+    * **json** - *(optional)* A JSON serializable object to include in the body
+    of the request.
+    * **headers** - *(optional)* Dictionary of HTTP headers to include on the
```

Comment:
Not sure if this is a typo?

```suggestion
    * **headers** - *(optional)* Dictionary of HTTP headers to include in the
```